apps/website/docs/try-it-out/try-it-out.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 displayed_sidebar: tryItOutSidebar
+hide_table_of_contents: true
```

Comment:
Allows to hide TOC there:

![image](https://user-images.githubusercontent.com/14183168/210520075-c7ca14d8-a6e8-4d94-84e6-414df199a10f.png)